activemq/datadog_checks/activemq/data/conf.yaml.example

```diff
@@ -13,7 +13,7 @@ instances:
     ## @param port - integer - required
     ## ActiveMQ port to connect to.
     #
-    port: 1099
```

Comment:
It appears this is the correct default port according to https://activemq.apache.org/jmx

No?

Reply:
Hmm, it does say that at the bottom, but every other example in the page uses 1616.

This SO answer uses yet another port too: https://stackoverflow.com/a/10368260